routes/admin.js

```diff
@@ -1,21 +1,18 @@
 const express = require('express')
 const router = express.Router()
 const passport = require('passport')
+const db = require('../database/db')
 
-router.get('/', function(request, response){
-  request.url < 2 ? response.render('admin') : response.redirect('/loggedout')
+router.post('/', function(request, response){
+  let { name, description, available, quantity, img } = request.body
+  db.createInventoryItem( name, description, available, quantity, img)
```

Comment:
See my suggested change in db - you could make this more concise with that change:
```js
db.createInventoryItem( request.body ) 
  .then( () => response.render( 'admin' ) )
```